optimistic-transaction.md

```diff
@@ -60,18 +61,16 @@ From the process of transactions in TiDB above, it is clear that TiDB transactio
 However, TiDB transactions also have the following disadvantages:
 
 * Transaction latency due to 2PC
-* In need of a centralized version manager
+* In need of a centralized timestamp allocation service
 * OOM (out of memory) when extensive data is written in the memory
 
-To avoid potential problems in application, refer to [transaction sizes](/transaction-overview.md#transaction-size) to see more details.
-
 ## Transaction retries
 
-TiDB uses optimistic concurrency control by default whereas MySQL applies pessimistic concurrency control. This means that MySQL checks for conflicts during the execution of SQL statements, so there are few errors reported in heavy contention scenarios. For the convenience of MySQL users, TiDB provides a retry function that runs inside a transaction.
+In the optimistic transaction model, transactions might fail to be committed because of write–write conflict in heavy contention scenarios. TiDB uses optimistic concurrency control by default, whereas MySQL applies pessimistic concurrency control. This means that MySQL adds locks during the execution of a SQL statement, and its Repeatable Read isolation level allows for non-repeatable reads, so commits generally do not encounter exceptions. To lower the difficulty of adapting applications, TiDB provides a retry function that runs inside the database.
```

Comment:
```suggestion
In the optimistic transaction model, transactions might fail to be committed because of write–write conflict in heavy contention scenarios. TiDB uses optimistic concurrency control by default, whereas MySQL applies pessimistic concurrency control. This means that MySQL adds locks during SQL execution, and its Repeatable Read isolation level allows for non-repeatable reads, so commits generally do not encounter exceptions. To lower the difficulty of adapting applications, TiDB provides an internal retry mechanism.
```